vhost-device-sound/src/lib.rs

```diff
@@ -183,6 +183,8 @@ pub enum Error {
     StreamWithIdNotFound(u32),
     #[error("Channel number not supported: {0}")]
     ChannelNotSupported(u8),
+    #[error("Could not disconnect stream")]
+    CouldNotDisconnectStream,
```

Comment:
Better put this in `stream::Error` since it exists already, instead of making this enum bigger:

https://github.com/dorindabassey/vhost-device/blob/1809dae37c873589e4954ab0fd24197b3080e30d/vhost-device-sound/src/stream.rs#L13-L24